webpack.common.config.js

```diff
@@ -78,22 +78,35 @@ SassdocPlugin.prototype.apply = compiler => {
       theme: 'herman',
       homepage: '/',
       descriptionPath: path.join(__dirname, 'STYLEGUIDE.md'),
+      groups: {
+        'config-color': 'Color Palettes',
+        'config-fonts': 'Webfonts',
+        'config-layout': 'Layout',
+        'config-scale': 'Sizes & Spacing',
+        icons: 'Icons',
+        sections: 'Page Sections',
+        typography: 'Typography',
+      },
       herman: {
         customCSS: cssPath,
         customHead:
           '<script src="https://use.typekit.net/slx1xnq.js"></script>' +
           '<script>try{Typekit.load({ async: true });}catch(e){}</script>',
         minifiedIcons: 'templates/_icons.svg',
         templatepath: path.join(__dirname, 'templates'),
+        displayColors: ['hex', 'hsl'],
         sass: {
           jsonfile: jsonPath,
           includepaths: [path.join(__dirname, 'static/sass')],
           includes: ['config/manifest'],
         },
         subprojects: [
           'accoutrement-color',
+          'accoutrement-init',
+          'accoutrement-layout',
           'accoutrement-scale',
           'accoutrement-type',
+          'sassdoc-theme-herman',
```

Comment:
Herman isn't currently setup to document itself as a subproject. Feel free to add an issue on Herman to fix that.